### PR TITLE
`<Table/>`- fixing table with action cell example

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
Fixing Table with ActionCell example to have a hover effect when hovering a table row.

### CODE REVIEW

File: src/Table/docs/examples/TableActionCellExample.js [hunk 1]
```diff
@@ -50,14 +50,15 @@ class TableActionCellExample extends React.Component {
                 ]}
                 // How many actions are visible, whereas the rest would be shown within a popover menu
                 numOfVisibleSecondaryActions={2}
-                alwaysShowSecondaryActions={false}
```
Comment (TalZa1810) [inline src/Table/docs/examples/TableActionCellExample.js]: it's false by default:
https://wix-style-react.now.sh/?activeTab=API&addons=0&full=0&panelRight=0&selectedKind=Components&selectedStory=TableActionCell&stories=1

File: src/Table/docs/examples/TableActionCellExample.js [hunk 2]
```diff
@@ -50,14 +50,15 @@ class TableActionCellExample extends React.Component {
...
               />
             ),
           },
         ]}
+        onRowClick={(rowData, rowNum) => console.log(`Row data is: ${JSON.stringify(rowData)}. Row num is: ${rowNum}`)}
```
Comment (nitayneeman) [inline src/Table/docs/examples/TableActionCellExample.js]: We should explain why this prop usage enables the hover

**Comment by nitayneeman** [comment]
After merging this PR, maybe it's worth to create an issue to separate the hover functionality from `onRowClick` - so that the row would have hover whether it's actually clickable or not.

**Comment by TalZa1810** [comment]
It is implemented according to the UX spec: https://zpl.io/bo8Kl7v

**Pull request merged**